fiftyone/utils/voc.py

```diff
@@ -183,6 +183,7 @@ def setup(self):
                 for p in etau.list_files(self.labels_path, recursive=True)
             }
         else:
+            logger.info("No labels found at %s" % self.labels_path)
```

Comment:
I'm planning a pass over all importers to handle these logging type messages. So perhaps just remove for now for consistency with the others?

The complexity was that I was trying to support unlabeled splits as well, where you wouldn't necessarily want to see this logging message because you in fact did not specify a labels path (but this is printed due to class internal defaults)

Reply:
Sounds good